logchange-core/src/test/resources/ErrorInConfigTest/changelog/unreleased/test-task.yml

```diff
@@ -0,0 +1,24 @@
+title: Test title 1
+authorss:
+  - nick: marwin1991
+    url: https://github.com/marwin1991
+merge_requestss:
+  - 1
+  - 2
+issues:
+  - 1
+links: # links to external systems, like redmine something else
+  - name: TASK_NUMBER
+    url: https://www.google.pl
+type: fixed # [added/changed/deprecated/removed/fixed/security/dependency_update/other]
```

Comment:
@NikodemMarek can we add one more test with type f.e. `fix` (it should not match available)